packages/protocol/contracts/L1/libs/LibProving.sol

```diff
@@ -196,8 +196,6 @@ library LibProving {
                     prover: msg.sender,
                     isContesting: evidence.tier == tran.tier
                         && tier.contestBond != 0,
-                    usingBlob: blk.usingBlob,
-                    blobHash: blk.blobHash,
```

Comment:
`evidence.blobHash` is passed in here, but it doesn't seem to be verified anywhere that it is indeed the same `blobHash` that was hashed inside `blk.metaHash`. To do that, the prover will have to provide all the meta data again, and then that data needs to be hashed again and checked that the hash matches the one in `blk.metaHash`.

Reply:
Good finding. Dani, maybe we should keep the code as-is if  passing in metadata as one additional parameter then hash it will probably cost more gas then storing these two variables (after ring buffer loops)?